packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -0,0 +1,273 @@
+                            >
+                                Run query
+                            </Button>
+                        </Tooltip>
+                    </Group>
+                </Group>
+            </Paper>
+            <Paper
+                shadow="none"
+                radius={0}
+                p="none"
+                withBorder
+                style={{ flex: 1 }}
+            >
+                <AceEditor
+                    mode="sql"
+                    theme="github"
+                    value={'SELECT * FROM table;'}
+                    height="100%"
+                    width="100%"
+                    editorProps={{ $blockScrolling: true }}
+                    enableBasicAutocompletion
+                    enableLiveAutocompletion
+                    wrapEnabled={true}
+                />
+            </Paper>
+            <ResizableBox
+                height={resultsHeight}
+                minConstraints={[50, 50]}
+                maxConstraints={[Infinity, maxResultsHeight]}
+                resizeHandles={['n']}
+                axis="y"
+                handle={
+                    <Divider
+                        h={3}
+                        bg="gray.3"
+                        pos="absolute"
+                        top={-2}
+                        left={0}
+                        right={0}
+                        sx={{
+                            cursor: 'ns-resize',
+                        }}
+                    />
+                }
+                style={{
+                    position: 'relative',
+                    display: 'flex',
+                    flexDirection: 'column',
+                }}
+                onResizeStop={(e, data) => setResultsHeight(data.size.height)}
+            >
+                <Paper shadow="none" radius={0} px="md" py="sm" withBorder>
+                    <Group position="apart">
+                        <Title order={5} c="gray.6">
+                            Results/Chart panel
+                        </Title>
+                        <Group spacing="md">
+                            <Tooltip
+                                variant="xs"
+                                label={
+                                    isResultsHeightMoreThanHalf
+                                        ? 'Collapse'
+                                        : 'Expand'
+                                }
+                                position="bottom"
+                            >
+                                <ActionIcon
+                                    size="xs"
+                                    onClick={() =>
+                                        setResultsHeight(
+                                            isResultsHeightMoreThanHalf
+                                                ? MIN_RESULTS_HEIGHT
+                                                : maxResultsHeight,
```

Comment:
Not sure if we can decrease the `maxResultsHeight` to be something _smaller_ so that we can still see some of the sql when we expand the results.

Reply:
My interpretation from the wireframe and description was that the SQL panel collapses completely. Once the developer has the correct SQL he collapses the SQL panel and the screen would be all for the chart and chart config. 
We will adjust and refine these details once we get user feedback.